### PR TITLE
remove(getEnv): unnecessary utility

### PR DESCRIPTION
**Pull request by Nciklol**

### CODE REVIEW

**Comment by Kurpp** [comment]
also you shouldnt throw errors if DISCORD_TOKEN is empty because people could name it something else!!!

**Comment by Nciklol** [comment]
> also you shouldnt throw errors if DISCORD_TOKEN is empty because people could name it something else!!!

great point, we should try to avoid forcing conventions as much as possible

**Review by Kurpp — CHANGES_REQUESTED** [review]

File: src/client/Client.ts [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 import type { Guild } from '#structures/Guild';
-import { getEnv } from '#utils/getEnv';
 import type { Channel } from '#structures/Channel';
 import { EventEmitter } from 'stream';
 import type { ClientOptions } from '../types/LIB';
 
 export class Client extends EventEmitter {
 	public gateway: Gateway;
-	public token: string = getEnv('DISCORD_TOKEN');
+	public token = process.env.DISCORD_TOKEN as string;
```
[the remark this answers is not in the document]
Comment (twisttaan) [reply src/client/Client.ts:9]: Good idea

**Review by twisttaan — APPROVED** [review]

**Review by Kurpp — APPROVED** [review]
LGTM

**Pull request merged**